docs/en/concepts-and-designs/scope-definitions.md

```diff
@@ -529,9 +529,9 @@ following **package**/**protocol** level metric contents.
 | destServiceInstanceName   | The dest pod name in kubernetes.                                   |           | string |
 | destLayer                 | The dest layer service in kubernetes.                              |           | string |
 
-#### Endpoint and Endpoint Relation
+#### Endpoint 
```

Comment:
`DefaultScopeDefine#K8S_ENDPOINT_REALATION(ID=77)` should be removed.

Reply:
Done.